main.gms

```diff
@@ -478,42 +482,47 @@ $setGlobal c_regi_capturescen  all !! def = all
 $setGlobal c_regi_synfuelscen  all !! def = all
 $setGlobal c_regi_sensscen  all !! def = all
 
+
+
 cm_TaxConvCheck = 0; !! def 1, which means tax convergence check is on
 																	  
-cm_biotrade_phaseout = 0; !! def 0
```

Comment:
this is minor, but I wouldn't commit in the main gms any change on switches initializations to numbers different from the default values. I don't mind if you want to keep those, but I think it is good practice to keep those unchanged and avoid everybody just overwriting those over and over in every commit without necessity.

Reply:
Ye, slips through when I add chunks with ``git add -pu``